src/lib/layers/editable-geojson-layer.js

```diff
@@ -501,13 +518,19 @@ export default class EditableGeoJsonLayer extends EditableLayer {
         drawFeature = bboxPolygon([minX, minY, maxX, maxY]);
       } else if (mode === 'drawCircleFromCenter') {
         const centerCoordinates = ((selectedFeature.geometry.coordinates: any): Array<number>);
-        const radius = Math.max(distance(selectedFeature, groundCoords || center), 0.001);
+        const radius = Math.max(
+          distance(selectedFeature, groundCoords || centerCoordinates),
```

Comment:
I fixed this already, You need to sync the code

Reply:
Fix this.

Reply:
Updated